blogs/2025/07/16/copilot-coding-agent.md

```diff
@@ -0,0 +1,104 @@
+---
+Order: 103
+TOCTitle: "Command GitHub's Coding Agent from VS Code"
+PageTitle: "Command GitHub's Coding Agent from VS Code"
+MetaDescription: VS Code's integration with GitHub Copilot Coding Agent allows you to delegate tasks to the agent and let it handle them in the background.
+MetaSocialImage: copilot-wip.png
+Date: 2025-7-16
+Author: Burke Holland
+---
+
+# Command GitHub's Coding Agent from VS Code
+
+July 16, 2025 by Burke Holland, [@burkeholland](https://twitter.com/burkeholland)
+
+Today, we’re excited to give you a first look at the Copilot Coding Agent in Visual Studio Code.
+
+Instead of only having one synchronous agent in VS Code, you can now have multiple agents running at once working on your behalf. This is _actually_ being a 10x developer. It's officially a thing. You get to hand off any work you need done to a squad of AI teammates.
+
+In this post, we'll take a look at what the Coding Agent is, give you an exclusive preview of the integrations coming to VS Code and even tell you how you can start using them today.
+
+## What is GitHub Copilot Coding Agent?
+
+[GitHub Copilot Coding Agent](https://github.blog/changelog/2025-07-14-start-and-track-github-copilot-coding-agent-sessions-from-visual-studio-code/) is an autonomous AI developer you can assign to any GitHub issue. The agent runs in GitHub and is fully integrated as a member of your repo that can comment on issues, open PRs, do code reviews and more. The agent can tackle complex work, implement features across your codebase, and even use Model Context Protocol (MCP) tools to work with databases or cloud services.
+
+1. You [enable the agent in your settings](https://docs.github.com/en/copilot/how-tos/agents/copilot-coding-agent/enabling-copilot-coding-agent).
+2. You assign an issue to Copilot.
+3. The agent does the work, opens a pull request, and keeps you in the loop.
+4. You review the PR, leave comments if you want changes, and the agent iterates until it’s done.
+
+![Copilot Coding Agent on GitHub](coding-agent-github.png)
+
+That's all there is to it. It's the workflow you already know with an agent in the mix.
+
+The agent runs within a temporary, isolated dev environment that gets spun up where the agent can explore the codebase, make changes, build the code, run tests, etc. - a complete dev enviornment just for the agent so it can function in a fully autonomous manner.
```

Comment:
```suggestion
The agent runs within a temporary, isolated dev environment that gets spun up where the agent can explore the codebase, make changes, build the code, run tests, etc. - a complete dev environment just for the agent so it can function in a fully autonomous manner.
```